frontend/src/metabase/collections/containers/CollectionSidebar.jsx

```diff
@@ -126,6 +128,23 @@ class CollectionSidebar extends React.Component {
             {t`View archive`}
           </Link>
         </Box>
+      </React.Fragment>
+    );
+  };
+
+  render() {
+    const { allFetched } = this.props;
+
+    return (
+      <Sidebar w={340} pt={3} data-testid="sidebar">
+        {allFetched ? (
+          this.renderContent()
+        ) : (
+          <div className="text-brand text-centered">
```

Comment:
Not a huge deal but it feels like it'd be nice to just pull out the existing loader UI elements into a shareable component and use it here.

Reply:
I've checked a few places where loaders are used, and they seem pretty different. Some components just render the `LoadingSpinner` without any wrappers and messages, and mostly all of them use different CSS classes for styling. I think we can get it wrong if we try to pull it out right now 🤔